src/Microsoft.VisualStudio.SlowCheetah.VS/Microsoft.VisualStudio.SlowCheetah.VS.csproj

```diff
@@ -7,6 +7,25 @@
     <Copyright>Copyright © Microsoft Corporation. All rights reserved.</Copyright>
     <CreateVsixContainer>false</CreateVsixContainer>
   </PropertyGroup>
+  
+  <ItemGroup>
```

Comment:
Does adding these files as Content cause them to also become resources in the assembly?